bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -260,6 +267,33 @@ impl Room {
         Ok(Timeline::new(timeline))
     }
 
+    pub async fn media_events_timeline(
+        &self,
+        filter: MediaEventsTimelineFilter,
+    ) -> Result<Arc<Timeline>, ClientError> {
+        let room = &self.inner;
+
+        let mut builder = matrix_sdk_ui::timeline::Timeline::builder(room);
+
+        builder = builder.event_filter(move |event, _| match event {
```

Comment:
Not sure it will change a thing, but it would be slightly safer to first filter with the [`default_event_filter`](https://github.com/matrix-org/matrix-rust-sdk/blob/46bdcd933a58c575f0fb9e0c8a65a90d3c14c29a/crates/matrix-sdk-ui/src/timeline/controller/mod.rs#L168).

Reply:
Oh good point, I'll add it just in case 👍